src/Data/Ord.purs

```diff
@@ -149,3 +151,13 @@ between low hi x
   | x < low = false
   | x > hi = false
   | true = true
+
+-- | The absolute value function. `abs x` is defined as `if x > zero then x
+-- | else negate x`.
+abs :: forall a. (Ord a, Ring a) => a -> a
+abs x = if x > zero then x else negate x
```

Comment:
I was about to say, isn't `negate x` just `zero - x`, but of course it isn't for `Int` and `Number` is it because of inlining! Yes, definitely.